pkg/executables/kubectl.go

```diff
@@ -695,6 +695,17 @@ func WithOverwrite() KubectlOpt {
 	return appendOpt("--overwrite")
 }
 
+func WithOutput(output string) KubectlOpt {
+	if output == "" {
+		return appendOpt()
```

Comment:
You can just return an empty func right? But this check is redundant because the user would not specify `WithOutput` if the string is empty.

Reply:
I am not sure whether the suggestion is to move the check on empty output to the client calling the function? Either ways, the check needs to happen somewhere, I have preferred it in here since the output from the customer might be empty and we can just return an empty func if that is the case.

Reply:
What I am saying is that the check is redundant and not needed because the user should not be specifying this option with an empty string, and if they did they may be on purpose. If you take a look at the others like `WithKubeconfig`, that's what we do.